plugins/card-resources/src/components/HomeCardPresenter.svelte

```diff
@@ -50,7 +55,9 @@
 
   $: socialId = message?.creator ?? card.modifiedBy
   $: personStore = getPersonByPersonIdStore([socialId])
-  $: person = $personStore.get(socialId)
+  $: personRef = card._class === contact.class.UserProfile ? (card as UserProfile).person : undefined
```

Comment:
Let's define `ObjectIcon` for `UserProfile` (you can check `Directs`)


So that there is no need to write special processing in each place